Backpack-SwiftUI/Card/Classes/BPKCard.swift

```diff
@@ -0,0 +1,115 @@
+/*
+ * Backpack - Skyscanner's Design System
+ *
+ * Copyright 2018-2022 Skyscanner Ltd
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+ 
+import SwiftUI
+
+public struct BPKCard<Content: View>: View {
+    public enum CornerStyle {
+        case large, small
+    }
+    
+    private var isSelected: Bool
+    private let content: Content
+    private var padded: Bool
+    private var cornerStyle: CornerStyle
+    private var backgroundColor: BPKColor = .backgroundTertiaryColor
+    private var tapAction : () -> Void = {}
+    
+    public init(
```

Comment:
do we have to add anything for accessibility here? how will this be announced/was anything custom done on the uikit version?